### DISCUSSION TITLE
player props not resetting on navigation in sveltekit

### DISCUSSION THREAD

**Discussion by rchrdnsh** [vidstack/player#894] [Q&A]
I have a vidstack player in a sveltekit app and it mostly works fine on initial load...but when i navigate away in the app and then navigate back again the values of the props are not reset...for example, it does not autoplay after navigation, it does not stay muted after navigation, etc...it's only when I refresh the page do the props reset correctly...

Any idea why this might be the case?

**Comment by mihar-22** [comment]
Hm, unfortunately not sure. We'll need to check if the issue persists with our library coming up for the next big release since we've rebuilt it from scratch.

**Reply by rchrdnsh** [reply]
ooooh…when is that release coming?

**Reply by mihar-22** [reply]
Soon, hard to say as we're rebuilding the site too. I was hoping end of this week but it might be more like end of next if things go well.

**Reply by rchrdnsh** [reply]
that's great, glad to hear it :-)

**Reply by ghost** [reply]
@mihar-22 any news on that big release?

**Reply by mihar-22** [reply]
Working on it! There's a lot to document, and building the new site has taken some time. If not this week, extremely like it'll be dropping next week.
